src/components/SecurityShell/SecurityShell.js

```diff
@@ -0,0 +1,264 @@
+  const [activeAction, setActiveAction] = useState(null);
+
+  return (
+    <div
+      className={`${namespace}__group ${securityShellNamespace}__group`}
+      {...other}
+    >
+      {Children.map(children, (child, index) =>
+        cloneElement(child, {
+          activeAction,
+          id: `${securityShellNamespace}__header__action--${index}`,
+          setActiveAction,
+        })
+      )}
+    </div>
+  );
+};
+
+const SecurityShellHeaderPopoverContent = ({ children, ...other }) => (
+  <div className={`${namespace}__popover__content`} {...other}>
+    {children}
+  </div>
+);
+
+const SecurityShellHeaderPopoverFooter = ({ children, ...other }) => (
+  <footer className={`${namespace}__popover__footer`} {...other}>
+    {children}
+  </footer>
+);
+
+const SecurityShellHeaderPopoverHeader = ({ children, ...other }) => (
+  <section className={`${namespace}__popover__header`} {...other}>
+    <span className={`${namespace}__popover__header__title`}>{children}</span>
+  </section>
+);
+
+const SecurityShellHeaderPopoverNotification = ({ ...other }) => (
+  <li className={`${namespace}__popover__list-item`}>
+    <HeaderNotification {...other} />
+  </li>
+);
+
+const SecurityShellHeaderPopoverNotifications = ({
+  ariaLabel,
+  children,
+  onClear,
+  title,
+  ...other
+}) => (
+  <>
+    <SecurityShellHeaderPopoverContent {...other}>
+      <span className={`${namespace}__popover__label`}>{title}</span>
+      <button
+        className={`${namespace}__popover__button`}
+        aria-label={ariaLabel}
+        onClick={onClear}
+      >
+        <Icon renderIcon={Close20} />
+      </button>
+    </SecurityShellHeaderPopoverContent>
+    <ScrollGradient color={theme.inverse02}>
+      <ul className={`${namespace}__popover__list`}>{children}</ul>
+    </ScrollGradient>
+  </>
+);
+
+const SecurityShellHeaderName = ({ children, offering, prefix, ...other }) => (
+  <Link className={`${namespace}__link`} {...other}>
+    {`${prefix} `}
+    <span className={`${namespace}__link__title--domain`}>
+      {`${offering} `}
+    </span>
+    <span className={`${namespace}__link__title--product`}>{children}</span>
+  </Link>
+);
+
+const SecurityShellReturnToBanner = ({ children, ...other }) => (
+  <Link
+    className={`${shellNamespace}__banner ${securityShellNamespace}__banner`}
+    style={{
+      backgroundImage: `url(${dataUri('../../images/aurora-banner@2x.png')})`,
+    }}
+    {...other}
+  >
+    <Icon
+      className={`${shellNamespace}__banner__icon`}
+      renderIcon={ArrowLeft16}
+    />
+    <span className={`${shellNamespace}__banner__text`}>{children}</span>
+  </Link>
+);
+
+const SecurityShellSkipToContent = ({ ...other }) => (
+  <div className={`${shellNamespace}__skip-to-content`}>
+    <Button
+      className={`${shellNamespace}__skip-to-content__link`}
+      // eslint-disable-next-line jsx-a11y/tabindex-no-positive
+      tabIndex={1}
+      {...other}
+    />
+  </div>
+);
```

Comment:
For a skip-to-content link the target `href` and the label text is going to be very important, so they shouldn't be relegated to a spread attribute.